src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -828,7 +828,7 @@ class VariantSampleMatrix[RPK, RK, T >: Null](val hc: HailContext, val metadata:
 
           annotateLoci(ord, finalType, inserter, product = product)
 
-        case Array(TInterval(_)) if vSignature == TVariant(GenomeReference.GRCh37) =>
+        case Array(TInterval(_, _)) if vSignature == TVariant(GenomeReference.GRCh37) || vSignature == !TVariant(GenomeReference.GRCh37) =>
```

Comment:
would this be easier to read as `vSignature.isOfType(TVariant(GenomeReference.GRCh37))`?

Reply:
Yup! I bet there are more of these floating around somewhere. I think I did this a bunch before realizing that I should probably define a function to do it.